manifest.json

```diff
@@ -1,7 +1,7 @@
 {
     "id": "templater-obsidian",
     "name": "Templater",
-    "version": "1.12.0",
+    "version": "1.13.0",
```

Comment:
don't update the version number in a feature PR, we'll do that when cutting a new release candidate. remove this from the PR.